crates/bevy_diagnostic/src/diagnostic.rs

```diff
@@ -41,6 +65,7 @@ pub struct Diagnostic {
     ema_smoothing_factor: f64,
     max_history_length: usize,
     pub is_enabled: bool,
+    pub num_of_decimals: DisplayPercision, // Seems like it needs to be usize
```

Comment:
I think the name `num_of_decimals` is clunky and does not clarify its intent. I prefer the name `precision` as mentioned in the original issue.